medusa/clients/torrent/generic.py

```diff
@@ -44,67 +44,30 @@ def __init__(self, name, host=None, username=None, password=None):
         self.username = app.TORRENT_USERNAME if username is None else username
         self.password = app.TORRENT_PASSWORD if password is None else password
         self.host = app.TORRENT_HOST if host is None else host
+        self.torrent_path = app.TORRENT_PATH if torrent_path is None else torrent_path
         self.rpcurl = app.TORRENT_RPCURL
         self.url = None
         self.response = None
         self.auth = None
-        self.last_time = time.time()
+        self.message = None
         self.session = ClientSession()
         self.session.auth = (self.username, self.password)
-
-    def _request(self, method='get', params=None, data=None, files=None, cookies=None):
-
-        if time.time() > self.last_time + 1800 or not self.auth:
-            self.last_time = time.time()
-            self._get_auth()
-
-        text_params = str(params)
-        text_data = str(data)
-        text_files = str(files)
-        log.debug(
-            '{name}: Requested a {method} connection to {url} with'
-            ' params: {params} Data: {data} Files: {files}', {
-                'name': self.name,
-                'method': method.upper(),
-                'url': self.url,
-                'params': text_params[0:99] + '...' if len(text_params) > 102 else text_params,
-                'data': text_data[0:99] + '...' if len(text_data) > 102 else text_data,
-                'files': text_files[0:99] + '...' if len(text_files) > 102 else text_files,
-            }
-        )
-
-        if not self.auth:
-            log.warning('{name}: Authentication Failed', {'name': self.name})
+        self.verify = certifi.where()
```

Comment:
Although I understand that it's good to verify certificates, in this case we dont know what torrent clients with what possible reverse proxies a user is using. Changing from verify=False to verify=certifi.where() can be a breaking change for users. I'm kinda in the middle of this. @medariox what do you think?

Maybe then it's better to utilize the setting `app.TORRENT_VERIFY_CERT`

Reply:
If someone wants to use a client with an ssl connection (https) and you keep it as verify=false you actually punch a hole in the ssl security because you are saying that everything is always ok.
This creates false security.
If you don't want ssl verification you should prohibit the use of https instead of saying everything is always ok by the 
verify=False statement.

Just my two cents